datadog/resource_datadog_integration_pagerduty.go

```diff
@@ -18,10 +18,16 @@ func resourceDatadogIntegrationPagerduty() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
+			"individual_services": {
```

Comment:
Shoudln't `is_individual` be a better name?

Reply:
Well this would be a good name on the actual service, but this attribute expresses that this resource *is using* individual services, so I'd prefer to keep it.

Reply:
ok